packages/rum-core/src/rumEvent.types.ts

```diff
@@ -1,4 +1,4 @@
-/* tslint:disable */
+/* eslint-disable @typescript-eslint/array-type */
```

Comment:
This file should not be modified by hand, run `node scripts/generate-schema-types.js` to generate it